content/en/serverless/azure_app_services.md

```diff
@@ -127,12 +127,20 @@ Learn more about [custom metrics][16].
 
 ## Troubleshooting
 
-To start troubleshooting your application, try these steps:
+1. If you are missing metrics and metadata in the APM trace panel and service page:
 
-1. Verify you've set `DD_SITE` and `DD_API_KEY` correctly.
-2. Do a full stop and start of your application.
-3. If not resolved, try uninstalling the extension and re-installing (this also ensures you are running the latest version).
-4. Still need help? Contact [Datadog support][17].
+It is likely that you do have the Azure Integration configured to monitor your application. Proper configuration will improve your ability to correlate metrics, traces, and logs in the Datadog platform, and avoid broken or empty screens. To fix this:
```

Comment:
```suggestion
It is likely that you do not have the Azure integration configured to monitor your application. Proper configuration improves your ability to correlate metrics, traces, and logs in the Datadog platform, as well as to avoid broken or empty screens. To fix this:
```